src/absenceDays/index.ts

```diff
@@ -1,24 +1,30 @@
 import BaseApi from '../baseApi';
 import responseHandler from '../utils/responseHandlers';
+import RequestParams from '../utils/requestParams';
 import { AbsenceDay } from './types';
 import { ReadParams } from '../utils/types';
 import { AxiosRequestConfig } from 'axios';
 
 export default class AbsenceDays extends BaseApi {
-  static resourceName = 'absencesDays';
+  private readonly resourceName = 'absencesDays';
```

Comment:
the same change needs to be applied to all resource classes

Reply:
This will be done in an extra PR because this is just a PoC